themes/default/content/blog/docker-build/index.md

```diff
@@ -0,0 +1,744 @@
+    tags: ["myregistry.com/user/repo:latest", "local-tag"],
+    push: true,
+    dockerfile: {
+        location: "./Dockerfile",
+    },
+    context: {
+        location: "../app",
+    },
+    target: "mytarget",
+    buildArgs: {
+        MY_BUILD_ARG: "foo",
+    },
+    cacheFrom: [{ registry: { ref: "myregistry.com/user/repo:cache" } }],
+    cacheTo: [{ registry: { ref: "myregistry.com/user/repo:cache" } }],
+    secrets: {
+        mysecret: "value",
+    },
+    addHosts: ["metadata.google.internal:169.254.169.254"],
+    ssh: {
+        default: ["/home/runner/.ssh/id_ed25519"],
+    },
+    network: "host",
+    platforms: ["linux/amd64"],
+    registries: [{
+        address: "myregistry.com",
+        username: "username",
+        password: pulumi.secret("password"),
+    }],
+});
+
+```
+
+### Migrating from Pulumi Docker v4
+
+The new Docker-Build provider largely exposes the same fields as v4 but
+pluralized or renamed to better align with the Docker CLI.
+
+It's important to note that the new Docker-Build provider matches the Docker
+CLI's behavior and _does not push_ images by default. If you want to push to a
+registry, you should include `push: true` just as you would include `--push` on
+the command line.
+
+```typescript
+// v4 Image
+const v4 = new docker.Image("v4-image", {
+    imageName: "myregistry.com/user/repo:latest",
+    skipPush: false,
+    build: {
+        dockerfile: "./Dockerfile",
+        context: "../app",
+        target: "mytarget",
+        args: {
+            MY_BUILD_ARG: "foo",
+        },
+        cacheFrom: {
+            images: ["myregistry.com/user/repo:cache"],
+        },
+        addHosts: ["metadata.google.internal:169.254.169.254"],
+        network: "host",
+        platform: "linux/amd64",
+    },
+    buildOnPreview: true,
+    registry: {
+        server: "myregistry.com",
+        username: "username",
+        password: pulumi.secret("password"),
+    },
+});
+
+// v4 Image after migrating to docker-build.Image
+const v4Migrated = new dockerbuild.Image("v4-to-buildx", {
+    tags: ["myregistry.com/user/repo:latest"],
+    push: true,
+    dockerfile: {
+        location: "./Dockerfile",
+    },
+    context: {
+        location: "../app",
+    },
+    target: "mytarget",
+    buildArgs: {
+        MY_BUILD_ARG: "foo",
+    },
+    cacheFrom: [{ registry: { ref: "myregistry.com/user/repo:cache" } }],
+    cacheTo: [{ registry: { ref: "myregistry.com/user/repo:cache" } }],
+    addHosts: ["metadata.google.internal:169.254.169.254"],
+    network: "host",
+    platforms: ["linux/amd64"],
+    registries: [{
+        address: "myregistry.com",
+        username: "username",
+        password: pulumi.secret("password"),
+    }],
+});
+
+```
+
+[BuildKit]: https://docs.docker.com/build/buildkit/
+[Pulumi Docker]: https://www.pulumi.com/registry/packages/docker/
+[inline cache]: https://docs.docker.com/build/cache/backends/inline/
+
+## Conclusion
```

Comment:
I think we can be more "pragmatic" here - restate the intro value props:
1. New package
2. Replacing the `docker.Image` resource
3. Brings new support for the latest Docker features
4. Multi-arch builds
5. Caching
6. Docker Build Cloud